invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/state/reducers/deposit.js

```diff
@@ -333,6 +334,15 @@ const depositReducer = (state = {}, action) => {
           },
         };
       }
+      if (!isRecordPublic && !isCommunityRestricted) {
```

Comment:
NOTE:
this PR fixes the problem by setting the record access setting back to public/public. it could be that it would be better to unblock the visibility setting. that should be discussed somehow!